coderd/audit.go

```diff
@@ -79,7 +79,7 @@ func (api *API) auditLogs(rw http.ResponseWriter, r *http.Request) {
 
 func (api *API) generateFakeAuditLog(rw http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
-	if !api.Authorize(r, rbac.ActionRead, rbac.ResourceAuditLog) {
+	if !api.Authorize(r, rbac.ActionCreate, rbac.ResourceAuditLog) {
```

Comment:
I might have missed it, but I don't see us enforcing this authorization in production. Is it sensible here? I imagine auditing shouldn't be user-permission dependent?

Reply:
If this is only used in unit tests then it probably doesn't matter. I was just fixing the auth check to match the actual action

Reply:
That makes sense!